src/pages/students/index.tsx

```diff
@@ -16,19 +16,19 @@ const Students = () => {
     onError: (errortraits) => {
       // errortraits allows one to access validation, code, message, and path
       // JSON.parse is utilized because errortraits.message is a string
-      const formattederrortraits = JSON.parse(errortraits.message) as {
-        validation: string;
-        code: string;
-        message: string;
-        path: string[];
-      }[];
+      try {
+        const formattederrortraits = JSON.parse(errortraits.message) as {
+          validation: string;
+          code: string;
+          message: string;
+          path: string[];
+        }[];
 
-      if (formattederrortraits[0].message == "Invalid email") {
-        alert("The provided email is in the incorrect format- please edit.");
-      }
-
-      //  can later insert other error messages here, as needed
-      else {
+        if (formattederrortraits[0].message == "Invalid email") {
+          alert("The provided email is in the incorrect format- please edit.");
+        }
```

Comment:
I would recommend adding an else statement here for if there's an error that matches the format you're checking against, but the error doesn't match the message you're checking against. Basically just something like 

else {
    alert("Something went wrong");
}

This is another thing that I think is probably good practice rather than absolutely essential given that the current try/catch code that you have is capturing all of the error contingencies we're currently aware of.